pkg/kubectl/cmd/get/get_test.go

```diff
@@ -317,6 +317,7 @@ func TestGetMultipleResourceTypesShowKinds(t *testing.T) {
 	expected := `NAME      READY   STATUS   RESTARTS   AGE
 pod/foo   0/0              0          <unknown>
 pod/bar   0/0              0          <unknown>
+
```

Comment:
this test should be updated to check the stderr output and verify nothing was printed to it

Reply:
e.g. 
```
streams, _, buf, bufErr := genericclioptions.NewTestIOStreams()
...
if e, a := "", bufErr.String(); e != a {
  t.Errorf("expected\n%v\ngot\n%v", e, a)
}
```

Reply:
SG. Thanks @liggitt !